controllers/topolvm_storageclass.go

```diff
@@ -93,7 +93,7 @@ func getTopolvmStorageClasses(lvmCluster *lvmv1alpha1.LVMCluster) []*storagev1.S
 	for _, deviceClass := range lvmCluster.Spec.DeviceClasses {
 		storageClass := &storagev1.StorageClass{
 			ObjectMeta: metav1.ObjectMeta{
-				Name: fmt.Sprintf("topolvm-%s", deviceClass.Name),
+				Name: fmt.Sprintf("odf-lvm-%s", deviceClass.Name),
```

Comment:
- with this change, we are make the project more downstream-ish, as we already have code to determine k8s environment (openshift or other), how about we use that info to decide on `odf-lvm-` vs `topolvm-`?

Reply:
I think a more generic sc name is probably fine for both upstream and downstream.

Reply:
ack, so we'll proceed with this name for now and later change it to generic then.